docs/UserGuide.md

```diff
@@ -174,6 +185,17 @@ Examples:
 * `add t/doctor n/Timmy-Tom c/98765432 s/neurology`
 * `add t/doctor n/John-White c/12312312 s/cardiology`
 
+### Display full details of a doctor: `view d/NAME`
+
+Displays the full details of a doctor
+
+Format: `view d/NAME`
+
+* `NAME` indicates the full name of the doctor, first name and last name are separated by `-`.
```

Comment:
Shall we have procedures on what happens if there are multiple people with the same name, for example, view using ID instead of name by showing the list of duplicate names first?

Reply:
Yes I agree that would be better to handle the case where different people have the same name